### DISCUSSION TITLE
Any ongoing work on migration v2 to .net8 ?

### DISCUSSION THREAD

**Discussion by karaahmed** [elsa-workflows/elsa-core#4661] [General]
Hi, 
Are there any ongoing efforts to migrate Elsa v2 to .Net8  ?

Thank you!

**Comment by GeorgievIt92** [comment]
Asking the same question. .Net8 seems much better.

**Reply by HubbuHubbu** [reply]
Hey there,

@karaahmed @GeorgievIt92 
the main developer of elsa made clear that this is not his top priority at the moment here:
https://github.com/elsa-workflows/elsa-core/issues/4667

But: this is an open source project and you can feel free to migrate elsa2 yourself to .net8, then create a pull request so the changes can be integrated.
